.github/workflows/backup-build.yml

```diff
@@ -2,9 +2,9 @@ name: Build and Test All
 
 on:
   push:
-    branches: [ master, test-ready ]
+    branches: [ master, test-ready, "bugfix-*" ]
```

Comment:
The `bugfix-*` looks like a duplicate of what I use in asadm and maybe uda/aql. If it is, I don't normally use them in this way. Normally I am not merging into bugfix but merging bugfix into master. I did not do that this time though

Reply:
Yeah I'll remove this so only PRs will trigger the runs